### PR TITLE
Add Contact descs to CommandTestUtil and SetPersonDescriptor tests

### PR DESCRIPTION
**Pull request by wrewsama**
Added valid and invalid use examples to `CommandTestUtil` for use in Command and CommandParser tests.
Also added tests for `SetPersonDescriptor`.

### CODE REVIEW

File: src/test/java/seedu/address/testutil/SetPersonDescriptorBuilder.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    }
+
+    /**
+     * Sets the slack Contact of the SetPersonDescriptor we are building.
+     */
+    public SetPersonDescriptorBuilder withSlack(String slack) {
+        Slack newSlack = new Slack(slack);
+        this.descriptor.setContacts(ContactType.SLACK, newSlack);
+        return this;
+    }
```
Comment (Tex-Tang) [inline src/test/java/seedu/address/testutil/SetPersonDescriptorBuilder.java:125]:
```
public SetPersonDescriptorBuilder withContact(ContactType type, String value) {
    this.descriptor.setContacts(type, value)
}
```
is more extensible.

**Review by Tex-Tang — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/address/testutil/SetPersonDescriptorBuilder.java (hunk 1 above)
Comment (wrewsama) [reply src/test/java/seedu/address/testutil/SetPersonDescriptorBuilder.java]: `SetPersonDescriptor::setContact` takes in a `Contact`, not a String so I might need to use a switch-case there. Is that still preferable over this?

Comment (wrewsama) [reply src/test/java/seedu/address/testutil/SetPersonDescriptorBuilder.java]: I see. Thanks for the tip!

**Review by sh4nH — APPROVED** [review]
LGTM

**Review by Tex-Tang — APPROVED** [review]
LGTM!

**Pull request merged**